recipes-dom0/dom0-image-thin-initramfs/files/meta-xt-prod-extra/recipes-bsp/u-boot/u-boot_%.bbappend

```diff
@@ -0,0 +1,36 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/files:"
+LICENSE = "CLOSED"
+
+UBOOT_MACHINE ?= "xenguest_arm64_defconfig"
+
+SRCREV = "${AUTOREV}"
+SRC_URI = "\
+    git://github.com/arminn/u-boot.git;protocol=https;branch=mr_rcar_pvblock; \
+"
+FILES_${PN} += " \
+      /xt/doma \
+"
+do_install() {
+  if [ -n "${UBOOT_CONFIG}" ]
```

Comment:
Either we use DomA or not some u-boot binary will be installed anyways (see DEPENDS which adds u-boot for all kinds of builds, even those which do not need u-boot at all). And the binary will go into /xt/doma/ folder. Som for example, a build with DomU Weston will have /xt/doma/u-boot.bin installed.

Reply:
now there is a coed that customize the installation folder
GUEST = "${@bb.utils.contains('XT_GUESTS_INSTALL', 'doma', 'doma', 'domu', d)}"

...
install -m 0744 ${D}/boot/u-boot-${GUEST}-${PV}-${PR}.${UBOOT_SUFFIX}  ${D}/xt/${GUEST}/${UBOOT_BINARY}